docs/source/use-bots.md

```diff
@@ -1,9 +1,11 @@
 # Dependencies updates in the repo
 
-When you install [Kebechet Bot][7] you can benefit from automatic pipelines and dependency management updates.
-You get automatic updates for your dependencies in case of CVE, new packages releases, performance changes.
-If you want to know more about Thoth bots and services have a look [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services).
+If you want to manage your project dependency updates automatically, you can install and set up the [Kebechet bot](https://github.com/thoth-station/kebechet) in your repo. Kebechet regularly checks if your project is using the optimal set of dependencies in terms of  CVE vulnerabilities, newer package releases, and performance changes. If it finds that the dependencies can be updated, it automatically opens a PR on your repo to make the required updates!
+
+An example of the Kebechet bot in action can be seen below.
 
 <div style="text-align:center">
 <img alt="Kebechet Automatic Update" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/KebechetAutomaticUpdate.png">
 </div>
+
+To learn more about Thoth bots and services, please check out the guide [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services).
```

Comment:
Can you add a "Next Step" with a link to the next notebook?

Reply:
+1 🙂

Reply:
Sure, added